data/styles/clear/include/Subways.mapcss

```diff
@@ -749,6 +749,12 @@ node|z14[railway=station][transport=subway][city=shiraz] {icon-image: subway-shi
 node|z15-[railway=station][transport=subway][city=shiraz] {icon-image: subway-shiraz-m.svg;}
 node|z16-[railway=subway_entrance][city=shiraz] {icon-image: subway-shiraz-s.svg;}
 
+/* Singapore Subway Station */
+node|z13[railway=station][transport=subway][city=singapore] {icon-image: subway-singapore-s.svg;icon-min-distance: 1;}
+node|z14[railway=station][transport=subway][city=singapore] {icon-image: subway-singapore-s.svg;}
```

Comment:
weird we have this z13 `icon-min-distance: 1;`  pattern throughout the file, it doesn't make any difference, but adds a redundant line to each city..

Reply:
i'll make a PR to remove them 👍️